x-pack/plugins/observability_solution/infra/public/components/autocomplete_field/suggestion_item.tsx

```diff
@@ -36,51 +37,51 @@ SuggestionItem.defaultProps = {
   isSelected: false,
 };
 
-const SuggestionItemContainer = euiStyled.div<{
+const SuggestionItemContainer = styled.div<{
   isSelected?: boolean;
 }>`
   display: flex;
   flex-direction: row;
-  font-size: ${(props) => props.theme.eui.euiFontSizeS};
-  height: ${(props) => props.theme.eui.euiSizeXL};
+  font-size: ${({ theme }) => theme.euiTheme.size.m};
```

Comment:
Issue: We're changing from a font size to a size token here. 
We should instead use `useEuiFontSize()` or `euiFontSize()`.

```tsx
font-size: ${() =>  useEuiFontSize('s').fontSize};

font-size: ${({ theme }) =>  euiFontSize(theme, 's').fontSize};
```